docs/roleplay/mastery.md

```diff
@@ -0,0 +1,135 @@
+- Use the `#instructors` channel in the Roleplay Hub to share training ideas, more detailed guides to specific lessons, and communicate students' progress.
+- When a student applies for mastery, reply to the application to confirm which lessons you have taught them and if you think they are ready to be a master.
+
+## Lesson Structure
+
+The first 5 lessons can be completed with a master *or* teacher *or* mix of both. The last 5 lessons must be completed with a master. [Water](https://avatar.fandom.com/wiki/Waterbending#Waterbending_master_level), [Earth](https://avatar.fandom.com/wiki/Earthbending#Earthbending_master_level), and [Fire](https://avatar.fandom.com/wiki/Firebending#Firebending_master_level) have moves classed as master level. Students can choose to learn two of these as part of their mastery training. Along with the outlines below, lessons can include sparring, training with other students, or visiting locations in order to learn a specific skill. Be creative!
+
+### Water
+Waterbending students can choose to follow either the traditional, or healing path to mastery. If choosing the healing path, offensive lessons will be replaced by healing lessons. For practical waterbending or history and culture, the lesson can relate specifically to the character's Clan if applicable.
+
+??? info "Lesson Structure"
+
+    1. Basics
+    2. Movement
+    3. Temperature control
+    4. Offensive skills 1 *or* Healing 1
+    5. Defensive skills 1
+    6. Practical waterbending or history and culture
+    7. Offensive skills 2 *or* Healing 2
+    8. Defensive skills 2
+    9. Master move 1
+    10. Master move 2
+
+### Earth
+
+??? info "Lesson Structure"
+
+    1. Strength and stability
+    2. Beginner offence and defence
+    3. History and culture
+    4. Intermediate offence
+    5. Intermediate defence
+    6. Practical earthbending
+    7. Advanced offence
+    8. Advanced defence
+    9. Master move 1
+    10. Master move 2
+
+### Fire
+
+??? info "Lesson Structure"
+
+    1. Breathing and heat
+    2. Beginner offence and defence
+    3. History and culture
+    4. Intermediate offence and defence
+    5. Energy, drive, and control
+    6. Practical firebending
+    7. Advanced offence
+    8. Advanced defence
+    9. Master move 1
+    10. Master move 2
+
+### Air
+
+Air master is an honourary position held by the most accomplished monks and nuns. It is achieved through intense training and hard work. Being a master is a gateway to more possibilities and opportunities. After completing the required lessons, there is a final trial. Students who pass this trial will receive their air mastery tattoos and be presented at a special ceremony by other air masters. Air masters must take on vows to teach others and respect the ways of masters.
```

Comment:
```suggestion
Air Master is an honourary position held by the most accomplished monks and nuns. It is achieved through intense training and hard work. Being a master is a gateway to more possibilities and opportunities. After completing the required lessons, there is a final trial. Students who pass this trial will receive their air mastery tattoos and be presented at a special ceremony by other air masters. Air masters must take on vows to teach others and respect the ways of masters.
```
Capitalizing the M to make it clear that Air Master is the title. Alternatively, you could also write *Air master* in italics to make that clear, though I prefer the former.